src/Processor/Gotenberg.php

```diff
@@ -157,6 +169,16 @@ public function getPdfFromString(string $html, array $params = [], bool $returnF
             $chromium->pdfFormat($params['pdfFormat']);
         }
 
+        if (empty($assets) === false) {
+            $assetStreams = [];
+
+            foreach ($assets as $asset) {
+                $assetStreams[] = Stream::path($asset['path'], $asset['filename']);
```

Comment:
LGTM, also not having any better ideas, maybe we should create a new function that copies a little `Mail::setAbsolutePaths` but it's more tied to the web2print usages.


So far, i am facing an issue, this file wold be unable to open and is a SEF instead of an absolute path.
![image](https://github.com/pimcore/web-to-print-bundle/assets/6014195/f0d466f6-0be8-4cd4-97e7-7ddabf8f8b6b)

Reply:
@matthiashamacher i am working on this again, did you have chance to look on this issue or should i take over and push changes over your PR?

Reply:
@kingjia90 Sorry, was on holidays and had no time to have a look. Feel free to take over 👍🏻